TCAT/Controllers/HomeViewController.swift

```diff
@@ -561,6 +567,17 @@ extension HomeViewController: WhatsNewDelegate {
         tableView.endUpdates()
     }
 
+    func dismissCardTemp() {
```

Comment:
`dismissCardTemp` and `unHideCard` do and undo the same action, right? Could you rename these to be the same? (e.g. `hideCard()`, `showCard()`, or `presentCard()`, `dismissCard()`). A `///` comment on the function would help explain this, I'd add something too